tests/private-active-active/main.tf

```diff
@@ -51,6 +51,7 @@ module "private_active_active" {
   redis_encryption_in_transit = true
   redis_use_password_auth     = true
   tfe_subdomain               = local.test_name
+  database_version            = 13
```

Comment:
This should be `postgres_engine_version`.

https://github.com/hashicorp/terraform-aws-terraform-enterprise/blob/main/variables.tf#L153-L157

Reply:
I made the change to add `postgres_engine_version = 13`